### PR TITLE
DOC: create custom events

### PR DESCRIPTION
**Pull request by stijnmoreels**
Relates to #46

### CODE REVIEW

File: docs/features/create-custom-events.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: "Create custom events"
+layout: default
+---
+
+## Create Custom Events
+Besides the [offcial Azure SDK event schemas](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.eventgrid.models?view=azure-dotnet), we support custom events also.
```
Comment (tomkerkhove) [inline docs/features/create-custom-events.md]: Let's highlight our work first over the built-in schemas to make it more discoverable
```suggestion
We allow you to create custom events next to the [official Azure SDK event schemas](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.eventgrid.models?view=azure-dotnet).
```

File: docs/features/create-custom-events.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+The official SDK provides a parent class called `EventGridEvent` which provides the data as an ordinary .NET `object`.
```
Comment (tomkerkhove) [inline docs/features/create-custom-events.md]: Same here, I would put our stuff first.

File: docs/features/create-custom-events.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+We provide another parent (which is the child of the official `EventGridEvent`) that provides your data as an typed instance instead.
```
Comment (tomkerkhove) [inline docs/features/create-custom-events.md]:
```suggestion
We provide a variation of the official `EventGridEvent` which provides your data as an typed instance instead.
```

File: docs/features/create-custom-events.md (hunk 2 above)
Comment (tomkerkhove) [inline docs/features/create-custom-events.md]:
```suggestion
The official SDK provides a class called `EventGridEvent` which provides your event data as an ordinary .NET `object`.
```

File: docs/features/create-custom-events.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        }
+
+        public NewCarRegistered(string id, string subject, string licensePlate) 
+            : base(id, subject, new CarEventData(licensePlate), DefaultDataVersion, DefaultEventType) 
+        {
+        }
+}
+```
+
+The typed data is now available as method called `.GetPayload()`:
```
Comment (tomkerkhove) [inline docs/features/create-custom-events.md]: Do we have docs on this somewhere else? I would rather point to that.
If not, we should add this and point to it there where `.Data` is also mentioned

Comment (stijnmoreels) [reply docs/features/create-custom-events.md]: 1. No, we don't have docs about `.Data`, so
2. I will add them to the `deserializing-events.md` file

**Pull request merged**